### DISCUSSION TITLE
#answer Mouse on amiberry menu not working

### DISCUSSION THREAD

**Discussion by wifi75** [BlitterStudio/amiberry#967] [Q&A] [answered]
Hello with the help of the previous experts in this group I managed to install amiberry on rpi4 with amibian lite without desktop.
The only thing that doesn't work and I don't know how to do it, is the mouse in the amiberry menu.
can someone help me?

**Comment by solskogen** [comment, marked as answer]
Check if you are in the 'input' group

**Reply by wifi75** [reply]
forgive me, but I don't understand what you mean

**Reply by solskogen** [reply]
When using KMS you need to be in a certain set of groups. At least 'render' and 'input' IIRC.
You have to add yourself to the file /etc/group or with usermod.

**Reply by wifi75** [reply]
I use fkms driver...

**Reply by solskogen** [reply]
Doesn't matter.

**Reply by wifi75** [reply]
I looked at the / etc / group file and I noticed that the items indicated to you before are already present ...
![image](https://user-images.githubusercontent.com/31816231/167829124-1e504bcc-0b1e-44fe-a22c-0cf060e9db88.png)
i have never done this thing, how should i change the entries?

**Reply by wifi75** [reply]
the weird thing is that the mouse pointer appears, but it doesn't move.
I repeat I am using OS lite

**Reply by solskogen** [reply]
Does it work if you run amiberry as root?

**Reply by wifi75** [reply]
yes

**Reply by solskogen** [reply]
Perhaps it's the tty group that you are missing? You have to try different groups, to see what's missing.

**Reply by wifi75** [reply]
Already present, I don't even know what to look for on the net to find a guide ...

**Reply by wifi75** [reply]
hi i don't know how to solve this problem, the groups indicated by you are all present.

**Reply by solskogen** [reply]
These are the groups I'm in:
`uid=1000(solskogen) gid=1000(solskogen) groups=1000(solskogen),4(adm),20(dialout),24(cdrom),27(sudo),29(audio),44(video),46(plugdev),60(games),100(users),104(input),106(render),108(netdev),997(gpio),998(i2c),999(spi)`

**Reply by solskogen** [reply]
But you didn't really read the FAQ, did you?

**Reply by wifi75** [reply]
hello in the faq I have not found anything about my problem.

**Reply by solskogen** [reply]
Q: How can I use Amiberry in KMSDRM (console) or Why is KMSDRM so slow?

**Reply by wifi75** [reply]
I downloaded the two files SDL2-2.0.22.zip  and libsdl2_2.0.22+dfsg-3.debian.tar.xz 
i unzipped sdl2 and inside this folder i unzipped the debian file then i ran the command:
dpkg-buildpackage -b -j$(nproc) -d

immediately after this error appears:

```
dpkg-buildpackage: info: host architecture arm64
 dpkg-source --before-build .
dpkg-source: info: using options from SDL2-2.0.22/debian/source/options: --extend-diff-ignore=debian/gbp.conf$
dpkg-source: warning: can't parse dependency clang:native
dpkg-source: warning: can't parse dependency clang:native
 debian/rules clean
```

then then after a few minutes at the end of the process I get this error:


```
make[2]: Leaving directory '/home/pi/tmp/SDL2-2.0.22'
make[1]: Leaving directory '/home/pi/tmp/SDL2-2.0.22'
make: *** [debian/rules:50: binary] Error 2
dpkg-buildpackage: error: debian/rules binary subprocess returned exit status 2
root@raspberry:/home/pi/tmp/SDL2-2.0.22#
```

so i can't find the two files
libsdl2-2.0-0_2.0.18_arm64.deb
libsdl2-dev_2.0.18_arm64.deb

what am I doing wrong?

**Reply by solskogen** [reply]
you have to post the real error.  Try building with `dpkg-buildpackage -b -d`

**Reply by wifi75** [reply]
Oh yess no it is ok after install deb file
sudo dpkg -i libsdl2-2.0-0_2.0.22+dfsg-3_arm64.deb libsdl2-dev_2.0.22+dfsg-3_arm64.deb

and 
apt install libsdl2-ttf-dev
apt install libsdl2-image-dev
mouse work! on amiberry gui!
very nice!!!
I'm happy!

please do not delete this post it is very important for the future.

Thank you sir!

**Reply by solskogen** [reply]
please mark this discussion as answered

**Reply by wifi75** [reply]
> please mark this discussion as answered

I didn't understand how to do it
